### DISCUSSION TITLE
[Feature Request] Table panel: API pagination

### DISCUSSION THREAD

**Discussion by lean257** [grafana/grafana#50047] [Dashboards, Panels & Visualizations]
The team from twinmaker plugin requested the ability to make paginated API calls. Right now our table pagination feature is making 1 query and paginate based on the given result, so this is purely frontend. But they want to be able to make multiple queries for large sets of data and display the result per page called. 

<img width="1086" alt="Screen Shot 2022-05-16 at 1 52 03 PM" src="https://user-images.githubusercontent.com/17167829/171505966-e63f2363-6e15-4202-b3eb-40499937ebdf.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
